template/files/routes.rb

```diff
@@ -18,6 +18,7 @@
 
   # Sidekiq Admin
   require 'sidekiq/web'
+  require 'sidekiq/cron/web'
   Sidekiq::Web.use Rack::Auth::Basic do |username, password|
     username == ENV['SIDEKIQ_USERNAME'] && password == ENV['SIDEKIQ_PASSWORD']
```

Comment:
This is vulnerable to timing attacks. Replace by
```
 Rack::Utils.secure_compare(::Digest::SHA256.hexdigest(username), ::Digest::SHA256.hexdigest(ENV["SIDEKIQ_USERNAME"])) &
      Rack::Utils.secure_compare(::Digest::SHA256.hexdigest(password), ::Digest::SHA256.hexdigest(ENV["SIDEKIQ_PASSWORD"]))
  end
```

as per the [README|https://github.com/mperham/sidekiq/wiki/Monitoring#standalone-with-basic-auth]

Probably in a separate PR as this one didn't touch the code, juste remove the duplicates

Reply:
Good pick, I prefer fixing this as part of the PR as it is located in a generator. Existing connectors may have to be updated.